improver_tests/standardise/test_RegridLandSea.py

```diff
@@ -70,10 +70,13 @@ class Test_process(IrisTest):
 
     def setUp(self):
         """Set up input cubes and landmask"""
+        # Domain corner of UK domain using 15.9,-2.5 as centre (as equalarea grid is configured)
```

Comment:
Might clarify comment to say it's defining a lat/lon grid that overlaps with (and completely covers) the UK standard grid for these tests.

Reply:
Update in commit bb66914d503803aedc364c469a53f559b0f52cc8.

Also updated grid spacing as I think with the default grid spacing of 10 it was creating a massive domain.